rules/coding-style/src/Rector/Variable/UnderscoreToPascalCaseVariableNameRector.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\Variable;
+
+use Nette\Utils\Strings;
+use PhpParser\Node;
+use PhpParser\Node\Expr\Variable;
+use PhpParser\Node\Identifier;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use Rector\Core\Util\StaticRectorStrings;
+
+/**
+ * @see \Rector\CodingStyle\Tests\Rector\Variable\UnderscoreToPascalCaseVariableNameRector\UnderscoreToPascalCaseVariableNameRectorTest
+ */
+final class UnderscoreToPascalCaseVariableNameRector extends AbstractRector
+{
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition('Change under_score names to pascalCase', [
+            new CodeSample(
+                <<<'PHP'
+final class SomeClass
+{
+    public function run($a_b)
+    {
+        $some_value = $a_b;
+    }
+}
+PHP
+,
+                <<<'PHP'
+final class SomeClass
+{
+    public function run($aB)
+    {
+        $someValue = $aB;
+    }
+}
+PHP
+
+            ),
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [Variable::class];
+    }
+
+    /**
+     * @param Variable $node
```

Comment:
@TomasVotruba  I changed @param 
```diff
     * @param Variable|PropertyProperty|PropertyFetch|StaticPropertyFetch $node
     * @param Variable $node
```
And now I see phpstan error
```
 ------------------------------------------------------------------------
 rules/coding-style/src/Rector/Variable/UnderscoreToPascalCaseVariableNameRector.php:78
 ------------------------------------------------------------------------
 - '#Property PhpParser\\Node\\Expr\\Variable\:\:\$name \(PhpParser\\Node\\Expr\|string\) does not accept PhpParser\\Node\\Identifier#'
 ------------------------------------------------------------------------
```
Can you advise what is the correct way to fix it?

Reply:
I think the message says to remove `new Identifier` and use bare string